src/lib/config/arbitrum.json

```diff
@@ -37,6 +37,10 @@
     "lidoRelayer": "",
     "balancerHelpers": "0x77d46184d22CA6a3726a2F500c776767b6A3d6Ab"
   },
+  "keys": {
+    "infura": "daaa68ec242643719749dd1caba2fc66",
+    "alchemy": "VBeQgTCRqqPtuuEPsFzRdwKXzDyN6aFh"
```

Comment:
The old key was incorrect, it was the Polygon key. This new one is the correct one for Arbitrum.